site/en/docs/extensions/whatsnew/index.md

```diff
@@ -232,17 +277,23 @@ associate results with the individual frames when injecting in multiple frames.
 
 ### Chrome 89: New API for managing tab groups {: #new-api-for-tab-groups-mv3-only }
 
+January 14, 2021
+
 The new [`chrome.tabGroups`](/docs/extensions/reference/tabGroups/) API and new capabilities in
 [`chrome.tabs`](/docs/extensions/reference/tabs/) let extensions read and manipulate tab groups.
 Requires Manifest V3.
 
 ### Chrome 89: Customizable permissions for web accessible resources {: #customizable-permissions-for-mv3-web-accessible-resources }
 
+December 23, 2020
+
 [Web accessible resources](/docs/extensions/mv3/manifest/web_accessible_resources/) definitions in
 Manifest V3 have changed to let extensions restrict resource access based on the requester's origin
 or extension ID.
 
-### 2021.04.08: Extension Manifest Converter {: #extension-manifest-converter}
+### Blog post: Extension Manifest Converter {: #extension-manifest-converter}
```

Comment:
```suggestion
### Blog post: extension Manifest Converter {: #extension-manifest-converter}
```

Reply:
I originally committed this change, but on closer look I reverted it because "Extension Manifest Converter" is a proper noun. As such, I think we should also change the name of the blog post from "Extension manifest converter" to "Extension Manifest Converter".

Reply:
Sound good.

Reply:
I think this is the only remaining issue. Let me know when you've changed the title.

Reply:
The most recent commit renames the blog post to "Extension Manifest Converter."